apps/web/src/pages/SearchRequests/SearchPage/components/SearchPools.tsx

```diff
@@ -58,28 +85,64 @@ const SearchPools = ({
           },
         )}
       </p>
+      <p data-h2-margin="base(x.5, 0, x1, 0)">
+        {" "}
```

Comment:
Is there a reason for this extra space?-

Reply:
nope.. left over .. will remove it. Thanks!